gensim/topic_coherence/indirect_confirmation_measure.py

```diff
@@ -126,24 +128,45 @@ def cosine_similarity(
         \vec{V}^{\,}_{m,\gamma}(W') =
             \Bigg \{{\sum_{w_{i} \in W'}^{ } m(w_{i}, w_{j})^{\gamma}}\Bigg \}_{j = 1,...,|W|}
 
-    Args:
-        segmented_topics: Output from the segmentation module of the
-            segmented topics. Is a list of list of tuples.
-        accumulator: Output from the probability_estimation module. Is an
-            accumulator of word occurrences (see text_analysis module).
-        topics: Topics obtained from the trained topic model.
-        measure (str): Direct confirmation measure to be used. Supported
-            values are "nlr" (normalized log ratio).
-        gamma: Gamma value for computing W', W* vectors; default is 1.
-        with_std (bool): True to also include standard deviation across topic
-            segment sets in addition to the mean coherence for each topic;
-            default is False.
-        with_support (bool): True to also include support across topic segments.
-            The support is defined as the number of pairwise similarity
-            comparisons were used to compute the overall topic coherence.
-
-    Returns:
-        list: of indirect cosine similarity measure for each topic.
+    Parameters
+    ----------
+    segmented_topics: list of (list of tuples)
+        Output from the segmentation module of the segmented topics.
+    accumulator: accumulator of word occurrences (see text_analysis module).
+        Output from the probability_estimation module. Is an topics: Topics obtained from the trained topic model.
+    measure : str
+        Direct confirmation measure to be used. Supported values are "nlr" (normalized log ratio).
+    gamma:
+        Gamma value for computing W', W* vectors; default is 1.
```

Comment:
`:math:` + no need to duplicate default value if it already defined OK in the signature (here and everywhere)